purldb-toolkit/tests/data/purlcli/expected_metadata_output_unique.json

```diff
@@ -0,0 +1,292 @@
+{
+    "headers": [
+        {
+            "tool_name": "purlcli",
+            "tool_version": "0.0.1",
+            "options": {
+                "command": "metadata",
+                "--purl": [
+                    "pkg:pypi/fetchcode",
+                    "pkg:pypi/fetchcode@0.3.0",
+                    "pkg:cargo/banquo",
+                    "pkg:nginx/nginx",
+                    "pkg:gem/rails",
+                    "pkg:rubygems/rails"
+                ],
+                "--file": null,
+                "--unique": true,
+                "--output": "<stdout>"
+            },
+            "purls": [
+                "pkg:pypi/fetchcode",
+                "pkg:pypi/fetchcode@0.3.0",
+                "pkg:cargo/banquo",
+                "pkg:nginx/nginx",
+                "pkg:gem/rails",
+                "pkg:rubygems/rails"
+            ],
+            "errors": [],
+            "warnings": [
+                "input PURL: 'pkg:pypi/fetchcode@0.3.0' normalized to 'pkg:pypi/fetchcode'",
```

Comment:
Please do NOT normalize unless someone requested it... and we surely should never drop a version when request for one.

Reply:
I get that.  You made the point over the weekend and I acknowleged and explained what I'm doing as a result.  Please look at the name of this file and the context -- this is the result when the `--unique` flag is included.